src/TraitsV2/MetaclassForTraits.class.st

```diff
@@ -53,6 +53,16 @@ MetaclassForTraits >> isClassTrait [
 	^ true
 ]
 
+{ #category : #testing }
+MetaclassForTraits >> isEmpty [ 
+	"Since we can have potential an empty array in the fluid definition (to be revisited) 
+	when the trait composition is aTrait it received the message isEmpty and would break
+	traitComposition 
+		ifNotNil: [ :t | t isEmpty ifFalse: [shiftClassBuilder traitComposition: traitComposition ]]."
+	^ false
```

Comment:
#isEmpty here looks really strange. It would be better to find different fix to the problem